cloud/aws/kops-asg.md

```diff
@@ -6,104 +6,55 @@ sidebar: home_sidebar
 redirect_from: "/cloud/aws/kops_asg.html"
 ---
```

Comment:
Add a short meta-desctiption describing the page.  This will help with SEO:

syntax

meta-description: "This page describes how to setup a production ready Portworx cluster using Kubernetes Kops. 
 You will learn how to dynamically provision persistent volumes using AWS Autoscaling groups to spin up EC2 instances."